<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SessionEditDialog</class>
 <widget class="QWidget" name="SessionEditDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>307</width>
    <height>431</height>
   </rect>
  </property>
  <layout class="QGridLayout" name="gridLayout_3">
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item row="3" column="0">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>6</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="6" column="0" colspan="3">
    <widget class="QWidget" name="terminalWidget" native="true">
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0" colspan="6">
       <widget class="KListWidget" name="m_terminalLBox">
        <property name="whatsThis">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="KPushButton" name="m_addTerminalButton">
        <property name="text">
         <string>&amp;Add...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1">
       <widget class="KPushButton" name="m_changeTerminalButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string>E&amp;dit...</string>
        </property>
       </widget>
      </item>
      <item row="1" column="2">
       <widget class="QToolButton" name="m_removeTerminalButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="toolTip">
         <string>Delete</string>
        </property>
        <property name="text">
         <string/>
        </property>
       </widget>
      </item>
      <item row="1" column="3">
       <spacer name="terminalSpacer">
        <property name="orientation">
         <enum>Qt::Horizontal</enum>
        </property>
        <property name="sizeType">
         <enum>QSizePolicy::Expanding</enum>
        </property>
        <property name="sizeHint" stdset="0">
         <size>
          <width>228</width>
          <height>20</height>
         </size>
        </property>
       </spacer>
      </item>
      <item row="1" column="4">
       <widget class="QToolButton" name="m_upTerminalButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item row="1" column="5">
       <widget class="QToolButton" name="m_downTerminalButton">
        <property name="enabled">
         <bool>false</bool>
        </property>
        <property name="text">
         <string/>
        </property>
        <property name="autoRepeat">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="sessionLabel">
     <property name="text">
      <string>&amp;Session name:</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
     <property name="buddy">
      <cstring>m_sessionName</cstring>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="2">
    <widget class="KLineEdit" name="m_sessionName">
     <property name="whatsThis">
      <string/>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QLabel" name="terminalsLabel">
     <property name="text">
      <string>Terminals:</string>
     </property>
     <property name="buddy">
      <cstring>m_terminalLBox</cstring>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutfunction spacing="KDialog::spacingHint" margin="KDialog::marginHint"/>
 <customwidgets>
  <customwidget>
   <class>KListWidget</class>
   <extends>QListWidget</extends>
   <header>klistwidget.h</header>
  </customwidget>
  <customwidget>
   <class>KPushButton</class>
   <extends>QPushButton</extends>
   <header>kpushbutton.h</header>
  </customwidget>
  <customwidget>
   <class>KLineEdit</class>
   <extends>QLineEdit</extends>
   <header>klineedit.h</header>
  </customwidget>
 </customwidgets>
 <tabstops>
  <tabstop>m_sessionName</tabstop>
  <tabstop>m_terminalLBox</tabstop>
  <tabstop>m_addTerminalButton</tabstop>
  <tabstop>m_changeTerminalButton</tabstop>
  <tabstop>m_removeTerminalButton</tabstop>
  <tabstop>m_upTerminalButton</tabstop>
  <tabstop>m_downTerminalButton</tabstop>
 </tabstops>
 <resources/>
 <connections/>
</ui>
